bin/crew

```diff
@@ -940,6 +940,23 @@ def post_install
     Dir.chdir post_install_tempdir do
       puts "Performing post-install for #{@pkg.name}...".lightblue
       @pkg.postinstall
+      if @pkg.gnome?
+        puts "Performing Gnome post-installs for #{@pkg.name}...".lightblue
+        # Is this needed here?
+        @device = JSON.parse(File.read("#{CREW_CONFIG_PATH}device.json"), symbolize_names: true)
```

Comment:
```ruby
@device = JSON.load_file("#{CREW_CONFIG_PATH}device.json", symbolize_names: true)
```

Reply:
My point was that this is already being done in crew at line 143:
```
@device = JSON.parse(File.read(CREW_CONFIG_PATH + 'device.json'), symbolize_names: true)
```
Should we change that to `load_file`?